ontology/tool/tool.ttl

```diff
@@ -493,10 +493,10 @@ tool:cpeid
 	.
 
 tool:creator
-	a owl:DatatypeProperty ;
+	a owl:ObjectProperty ;
 	rdfs:label "creator"@en ;
 	rdfs:comment "The creator organization for a particular tool."@en ;
-	rdfs:range xsd:string ;
+	rdfs:range core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the tool namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.